### ISSUE TITLE
How to add audio track on current (5.17) version?

### ISSUE THREAD

**Issue by pstrg** [RandomEngy/VidCoder#633]
I see that older versions used to have an "Add track" button for adding audio tracks:
![old](https://user-images.githubusercontent.com/13485213/78140615-abd66d00-7400-11ea-89f8-f825866c625c.jpg)
The current version doesn't have one and I have not found a way to do it.
![new](https://user-images.githubusercontent.com/13485213/78140822-fbb53400-7400-11ea-9ee2-80190feb88e6.jpg)

**Comment by RandomEngy** [comment]
In the new UI all the available audio tracks are listed and you can check the ones you want to include. If you want to include more than one copy, you can click on the "+" to duplicate it.

**Comment by pstrg** [comment]
Thank you.
But what I want to do is to add an (external) audio track that's not already included in the main file.
Is it possible to do that with VidCoder or should another tool be used?

**Comment by RandomEngy** [comment]
That wasn't possible with any version of VidCoder. You should look for a different tool to do that: a muxer or video editor.

VidCoder is a transcoder.

**Comment by bnwbass** [comment]
This is timely thread for me, I have over 400 titles in DVD structure that I need to encode mp4.
On the topic of audio, Media info indicates the source file audio as 
AC-3: Dolby Digital
Muxing mode  : DVD-Video
Bit rate     : 448 kb/s
Channel( : 6 channels
Channel layout  : L R C LFE Ls Rs
Compression mode  : Lossy

However, vicoder results output as only 2 ch stereo.
I check pass-through and this show's in the audio edit box
![image](https://user-images.githubusercontent.com/5840657/78294174-bc5d1500-74de-11ea-8d0a-8bae64f6d561.png)

**Comment by RandomEngy** [comment]
I think this a separate issue.

It doesn't look it's actually doing passthrough. What do your settings look like? Should be something like this:

![image](https://user-images.githubusercontent.com/9450861/78300676-b9692100-74ec-11ea-9bad-08ecf91f38d3.png)

**Comment by bnwbass** [comment]
Yes , you are correct my error. I did a test with pass through and all good.  Related question.
I have 400 titles to convert, from what I see I need to set passthru for each in edit., Is there a global setting that will apply every item in the Que?

**Comment by RandomEngy** [comment]
Should be an "Apply to queue" button:

![image](https://user-images.githubusercontent.com/9450861/78307501-e7556200-74fa-11ea-99ec-a9b4f4c5b603.png)

Or right click on items and select this option:
![image](https://user-images.githubusercontent.com/9450861/78307548-0227d680-74fb-11ea-941a-b211afbfa1e5.png)
